libs/application/templates/secondary-school/src/fields/overview/CustodianOverview.tsx

```
@@ -44,47 +45,46 @@
           ? overview.custodian.subtitle
           : overview.otherContact.subtitle,
       )}
       isEditable={application.state === States.DRAFT}
     >
       <Box>
-        {!!parents.length && (
+        {!!custodiansExternalData.length && (
           <GridRow>
-            {parents.map((parent, index) => (
+            {custodiansExternalData.map((custodian, index) => (
               <GridColumn span="1/2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)